relay/workflow.py

```diff
@@ -1413,97 +1341,68 @@ async def _set_device_info(self, target, field, value):
     #     await self.sendReceive(event)
     #     # await self._set_device_info(target, )
 
-    def led_info(self, rotations:int=None, count:int=None, duration:int=None, repeat_delay:int=None, pattern_repeats=None, colors=None):
```

Comment:
was the intention to remove this method from the relay object?  or just remove the dependence on 'self'?

you could do the former with the @staticmethod decorator (https://docs.python.org/3/library/functions.html#staticmethod)